ui-frontend/packages/catalog-ui-search/src/main/webapp/component/result-selector/query-feed.tsx

```diff
@@ -275,15 +276,31 @@ const QueryFeed = ({ selectionInterface }: Props) => {
           >
             {({ handleClick }) => {
               return (
-                <Button
-                  data-id="heartbeat-button"
-                  onClick={handleClick}
-                  className="details-view is-button"
-                  title="Show the full status for the search."
-                  data-help="Show the full status for the search."
-                >
-                  <span className="fa fa-heartbeat" />
-                </Button>
+                <div>
+                  <div style={{ position: 'relative' }}>
```

Comment:
Try and use the tailwind utilities for these more common stylings.

In this case it would be:
```
className="relative"
```